_layouts/get-started.html

```diff
@@ -12,12 +12,14 @@ <h2 class="step">Step <span class="step-no">1</span></h2>
     </div>
     <div class="width-10-12 width-12-12-m">
       <h3>Get Infinispan</h3>
-      <p>Add a username and password, then run the one of the following commands:
+      <p>Important: Authentication and Authorization are enabled by default. Use `admin` user name
+      to implicitly have a user with the admin role.</p>
+      <p>Run the one of the following commands:
         <pre><code>
-podman run -it -p 11222:11222 -e USER="username" -e PASS="password" --net=host quay.io/infinispan/server:{{site.data.ispn.stable.minor_version}}
+podman run -it -p 11222:11222 -e USER="admin" -e PASS="pass" --net=host quay.io/infinispan/server:{{site.data.ispn.stable.minor_version}}
```

Comment:
Is there a reason for changing password to pass? IMO "password" is a little bit more meaningful.